src/Security/Security.php

```diff
@@ -1086,7 +1086,7 @@ public static function findAnAdministrator()
     /**
      * Flush the default admin credentials
      *
-     * @deprecated 4.0.0:5.0.0 Please use DefaultAdminService::clearDefaultAdmin()
+     * @deprecated 4.0.0 Please use DefaultAdminService::clearDefaultAdmin()
```

Comment:
```suggestion
     * @deprecated 4.0.0 Use DefaultAdminService::clearDefaultAdmin() instead
```
Update the notice message and version as well.